core/cmd/remote_client_test.go

```diff
@@ -62,6 +62,8 @@ func startNewApplication(t *testing.T, setup ...func(opts *startOptions)) *cltes
 	// Setup config
 	config, cfgCleanup := cltest.NewConfig(t)
 	t.Cleanup(cfgCleanup)
+	config.Set("DEFAULT_HTTP_TIMEOUT", "30ms")
```

Comment:
As jobs are now actually executed in the RunJobV2 - we keep these much shorter to speed up the tests.